Base/res/themes/Miniminty.ini

```diff
@@ -0,0 +1,102 @@
+[Colors]
+DesktopBackground=#2f3436ff
+Accent=#3b9970
+ActiveWindowBorder1=#2b6e51
+ActiveWindowBorder2=#52d49b
+ActiveWindowTitle=white
+ActiveWindowTitleShadow=black
+ActiveWindowTitleStripes=#348562
+InactiveWindowBorder1=#8c8c8c
+InactiveWindowBorder2=#8c8c8c
+InactiveWindowTitle=#fafafa
+InactiveWindowTitleShadow=#8c8c8c
+InactiveWindowTitleStripes=#a0a0a0
+MovingWindowBorder1=#2b6e51
+MovingWindowBorder2=#52d49b
+MovingWindowTitle=white
+MovingWindowTitleShadow=#1a4231
+MovingWindowTitleStripes=#2b6e51
+HighlightWindowBorder1=#2b6e51
+HighlightWindowBorder2=#2b6e51
+HighlightWindowTitle=white
+HighlightWindowTitleShadow=#1a4231
+HighlightWindowTitleStripes=#2b6e51
+HighlightSearching=#ffff00
+HighlightSearchingText=black
+MenuBase=#f2fffc
+MenuBaseText=#0c1f16
+MenuStripe=#52d49b
+MenuSelection=#52d49b
+MenuSelectionText=black
+Window=#bee6ce
+WindowText=#0c1f16
+Button=#bee6ce
+ButtonText=#0c1f16
+Base=#f2fffc
+BaseText=#0c1f16
+DisabledTextFront=#999999
+DisabledTextBack=white
+ThreedHighlight=white
+ThreedShadow1=#97a89a
+ThreedShadow2=#252925
+HoverHighlight=#52d49b
+Selection=#52d49b
+SelectionText=#0c1f16
+InactiveSelection=#9e9e9e
+InactiveSelectionText=#0c1f16
+PlaceholderText=#999999
+RubberBandFill=#52d49b3c
+RubberBandBorder=#2b6e51
+Link=blue
+ActiveLink=red
+VisitedLink=magenta
+Gutter=#c7ebe1 
+GutterBorder=white
+Ruler=#e8f5f2
+RulerBorder=#cfe6d2
+RulerActiveText=#606060
+RulerInactiveText=#999999
+TextCursor=#153627
+FocusOutline=#909090
+SyntaxComment=#bcbdc4
+SyntaxNumber=#255e46
+SyntaxString=#4d944d
+SyntaxType=#36915a
+SyntaxPunctuation=black
+SyntaxOperator=black
+SyntaxKeyword=#1f5434
+SyntaxControlKeyword=#1f5434
+SyntaxIdentifier=#000000
+SyntaxPreprocessorStatement=#00a0a0
+SyntaxPreprocessorValue=#1a4231
+SyntaxFunction=#112e1c
+SyntaxVariable=#1c3275
+SyntaxCustomType=#77db9e
+SyntaxNamespace=#77db9e
+SyntaxMember=red
+SyntaxParameter=#53a053
+Tooltip=#ffffe1
+TooltipText=#0c1f16
+Tray=#cfe6d2
+TrayText=#ffffff
+
+[Alignments]
+TitleAlignment=Left
+
+[Flags]
+IsDark=false
+
+[Metrics]
+BorderThickness=4
+BorderRadius=0
+TitleHeight=19
+TitleButtonWidth=15
+TitleButtonHeight=15
+
+[Paths]
+ActiveWindowShadow=/res/icons/themes/Miniminty/frame-shadow-dark.png
+InactiveWindowShadow=/res/icons/themes/Miniminty/frame-shadow-light.png
+MenuShadow=/res/icons/themes/Miniminty/frame-shadow-light.png
+TaskBarShadow=/res/icons/themes/Miniminty/frame-shadow-light.png
+TooltipShadow=/res/icons/themes/Miniminty/frame-shadow-light.png
```

Comment:
Are these shadows modified in any way? If not, please use paths to the default theme here instead of duplicating the files.